### DISCUSSION TITLE
Azure Data Studio stuck on loading accounts screen - No browser pop up screen presented

### DISCUSSION THREAD

**Discussion by AzureDataStudio** [microsoft/azuredatastudio#25149] [General]
Hi Team, I have tried uninstalling, reinstalling app. Also updating app - nothing works. The linked accounts window gets stuck on "Loading accounts" page and doesn't pop up with browser window. Tested @builtin extension and it's enabled. Tried disabling, re-enabling but didn't help. 

One thing I have noticed is that it works under my account on same system but doesn't work for user. What should I check now?

**Comment by cheenamalhotra** [comment]
Please check your proxy settings and if they apply for the user account in question, you need to provide the same to allow network access via proxy. For more details on how to configure proxy URL, refer to: https://learn.microsoft.com/en-us/azure-data-studio/azure-connectivity#proxy-setup-for-azure-authentication

**Comment by AzureDataStudio** [comment]
There are no proxy settings in place.

**Comment by cheenamalhotra** [comment]
Can you gather and find any errors in [Azure logs](https://learn.microsoft.com/en-us/azure-data-studio/azure-connectivity#azure-logging-level)?

**Comment by AzureDataStudio** [comment]
Hi,

I have been able to resolve this issue so good to close this request.

It was to do with one of the security groups.


Thanks
Ranjan
________________________________
